Makefile

```diff
@@ -551,6 +551,7 @@ $(eval $(call build-package,ratelimit,0.0_git20230331-r0))
 $(eval $(call build-package,logstash,8.7.0-r0))
 $(eval $(call build-package,ntpd-rs,0.2.1-r0))
 $(eval $(call build-package,trust-dns,0.22.0-r0))
+$(eval $(call build-package,lvm2,2.03.20-r0))
```

Comment:
We're changing our build process a little, can you rebase and update `packages.txt` instead of updating the makefile? 🙏

Reply:
Fixed!